### DISCUSSION TITLE
Firewall rules.

### DISCUSSION THREAD

**Discussion by janssen-bart** [lithnet/access-manager#75] [Q&A]
Hi,

As (I hope ;-) ) most company's try to close down servers and have almost no internet access, it is maybe a good idea to provide a list in the documentation with URLS, ports to open. 

Kind regards
Bart Janssen

**Comment by ryannewington** [comment]
Thanks for the suggestion @janssen-bart .

The firewall requirements have now been documented here https://github.com/lithnet/access-manager/wiki/Firewall-rule-requirements
